x/staking/keeper/genesis.go

```
@@ -331,6 +360,32 @@
 		return nil, err
 	}
 
+	var initConsAddrs, rotatedConsAddrs []types.RotatedConsensusAddresses
+	err = k.ConsAddrToValidatorIdentifierMap.Walk(ctx, nil, func(newBz []byte, initBz []byte) (stop bool, err error) {
```

golangci-lint (1.62.2)

File is not `gofumpt`-ed with `-extra`

(gofumpt)